.github/ISSUE_TEMPLATE/project-kickoff.md

```diff
@@ -27,6 +27,8 @@ assignees: andreahewitt-odd, shiragoodman
 - DEPO Product Lead (required): **name**
 - Entire VFS team (recommended): **names**
 
+VSP will only invite those specified, so be sure to list your content, IA, accessibility and QA specialists, if applicable.
+
 **Do you anticipate this initiative requiring updates to and/or creation of unauthenticated static content pages?** YES/NO
```

Comment:
This is not something I would hope a VFS team would know this early. Instead I'd hope they determine the answer to this _after_ they've done discovery and a little bit of testing to determine the solution approach they want to take for their problem. I would expect them to know this by the time they're ready for design intent though. Could we consider moving the question to that template?

Reply:
Yes, I agree it should be moved. This will be reflected on the assessment checklist. Do you want to move it now or wait til the rough draft of the checklist is complete?